frontend/src/components/Project/GMS/GMSViewer.js

```diff
@@ -366,9 +366,10 @@ const GmsViewer = () => {
       </Tabs>
       <DownloadButton
         disabled={invalidInputs()}
-        downloadURL={
-          CONSTANTS.PROJECT_API_GMS_DOWNLOAD_ENDPOINT + "/" + downloadToken
-        }
+        downloadURL={CONSTANTS.PROJECT_API_GMS_DOWNLOAD_ENDPOINT}
+        downloadToken={{
+          gms_token: downloadToken,
+        }}
         fileName="Projects_Ground_Motion_Selection.zip"
```

Comment:
I believe the filenames are something that needs to be changed.
E.g., if you download Hazard Curve, Disaggregation includes the selected IM as is the IM the user chose but for UHS, because you only select return periods(now it's going to be an annual exceedance rate), which isn't ideal to be part of filenames. Also, GMS has more user inputs. Hence, we haven't decided what we want for UHS and GMS yet.